src/main/java/de/tum/in/www1/artemis/domain/TextExercise.java

```diff
@@ -25,6 +27,11 @@
     @JsonIgnore
     private List<TextCluster> clusters;
 
+    @ManyToOne
+    @JoinColumn(table = "text_exercise_details")
+    @JsonIgnoreProperties("exercises")
```

Comment:
```suggestion
    @JsonIgnore
```
Same Idea as in TextBlock